### PR TITLE
Implement tests ;; WLFE-26

### PR DESCRIPTION
**Pull request by ofrogon**
Also:
- Change the license to Apache
- Remove the Prettier tag on the REANDME since we don't use it
- Really fix the `prompts` folder typo
- Add minimal documentation and templates

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -3,4 +3,7 @@ foundry.iml
 
 node_modules
 
-dist
\ No newline at end of file
+dist
+
+# Jest
+jest.config.js
```
Comment (alexasselin008) [inline .eslintignore:9]: is this necessary? config files should still be linted?

File: packages/create-workleap/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+A CLI to create new Workleap projects using prompts and your favorite package manager `create` command.
+
+## Interactive
+
+You can create a new project interactively by running:
+
+```cmd
+npm create workleap@latest <output-directory>
+# or
+npm create-workleap@latest <output-directory>
```
Comment (alexasselin008) [inline packages/create-workleap/README.md:12]: `npm create-workleap@latest <output-directory>` is duplicated

File: .eslintignore (hunk 1 above)
Comment (ofrogon) [reply .eslintignore:9]: The config files normally follow the tool format convention, so I don't see advantage to have lint warnings on them.

File: packages/create-workleap/README.md (hunk 2 above)
Comment (alexasselin008) [inline packages/create-workleap/README.md:12]: npx instead of npm?

File: .eslintignore (hunk 1 above)
Comment (alexasselin008) [reply .eslintignore:9]: is it because it's CJS and eslint checks for ESM?

Comment (ofrogon) [reply .eslintignore:9]: Yes, for example if I activate the scan for the `jest.config.js` I got that kind of error:

```
C:\Dev\foundry\foundry\packages\create-workleap\jest.config.js
  1:1  error  'module' is not defined  no-undef
```

File: tsconfig.eslint.json [hunk 3]
```diff
@@ -3,5 +3,5 @@
   "compilerOptions": {
     "noEmit": true
   },
-  "include": ["src", "test", "tools", "bin"]
+  "include": ["src", "packages/*/tests"]
```
Comment (alexasselin008) [inline tsconfig.eslint.json:6]: ça pourrait pas être juste ça?

"include": [ "packages/*"]

File: .eslintignore (hunk 1 above)
Comment (alexasselin008) [reply .eslintignore:9]: Est-ce qu'on pourrait mettre ce fichier là en ts à la place? 

image
https://jestjs.io/docs/27.x/configuration

Comment (ofrogon) [reply .eslintignore]: Seems to work nicely 👍

File: tsconfig.eslint.json (hunk 3 above)
Comment (ofrogon) [reply tsconfig.eslint.json]: Maybe it requires revisiting some `tsconfig.json` file include/exclude scope, but just having `package/*` break eslint completely

**Review by alexasselin008 — APPROVED** [review]

**Pull request merged**